users/yet-another-developer/leader.c

```diff
@@ -0,0 +1,55 @@
+#include "leader.h"
+
+LEADER_EXTERNS();
+
+#define TAP_ONCE(code)  \
+  register_code (code); \
+  unregister_code (code)
+
+// Runs constantly in the background, in a loop.
+void matrix_scan_user(void){
+  static bool has_ran_yet;
+  if (!has_ran_yet) {
+    has_ran_yet = true;
+    startup_user();
```

Comment:
Not needed anymore.   You can use `keyboard_post_init_user` now.

Reply:
Is it safe to remove from `matrix_scan_user`? I checked your userspace `matrix_scan_user` and it still has the same code in `drashna.c`

Reply:
I've removed the code regardless, kindly review if changes are faultless. 
I've removed the code for both {user}.c and in leader.c under `matrix_scan_user`.

Reply:
It does?  Whoops, need to remove that too, then! :)